### PR TITLE
Change deprecation warning to link to `https://stylelint.io/awesome-stylelint#formatters`

### LINKED ISSUE #8111: Change `detail` sentence of `GITHUB_FORMATTER` deprecation warning
### What is the problem you're trying to solve?

A lot of linters have their own built-in GitHub formatter, and this is also the case with stylelint

However, the GitHub formatter has been deprecated and it suggests to switch to a third-party alternative which prints deprecation messages, it is not trivial to configure (apparently), and appears to not be maintained

### What solution would you like to see?

Reconsider the deprecation, or advise for an actively maintained alternative

The alternative,

- https://github.com/xt0rted/stylelint-actions-formatters

should be used in conjunction with (or entirely replaced by, this is not clear)

https://github.com/xt0rted/stylelint-problem-matcher

does not seem maintained. 

I didn't find a way to make it work in a GitHub action. I have some test cases here:

Ref: https://github.com/diowa/ruby3-rails7-bootstrap-heroku/pull/984

I've asked for some clarifications one month ago at xt0rted/stylelint-problem-matcher#592 and also commented at the original request to make it the official suggested formatter at https://github.com/xt0rted/stylelint-actions-formatters/issues/187#issuecomment-2399118878

---

https://github.com/stylelint/stylelint/blob/13933eda2c6430ea2c595f02f47799adbda771df/lib/standalone.mjs#L114

### PR DESCRIPTION
**Pull request by ryo-manba**
> Which issue, if any, is this issue related to?

Closes #8111 

> Is there anything in the PR that needs further explanation?

No, it's self-explanatory.

### CODE REVIEW

File: lib/standalone.mjs [hunk 1]
```diff
@@ -111,7 +111,7 @@ export default async function standalone({
 		emitDeprecationWarning(
 			'"github" formatter is deprecated.',
 			'GITHUB_FORMATTER',
-			'Please use "stylelint-actions-formatters" instead.',
+			'Please use "@csstools/stylelint-formatter-github" instead.',
```
Comment (romainmenke) [inline lib/standalone.mjs:114]: [question]

Should we maybe link to the list here: https://github.com/stylelint/awesome-stylelint/blob/main/README.md#formatters

Then users can pick the one they prefer and we won't need to update the message if a better formatter appears in the future.

Maybe something like:

```
See https://github.com/stylelint/awesome-stylelint/blob/main/README.md#formatters for alternative formatters
```

Comment (ryo-manba) [reply lib/standalone.mjs:114]: Good suggestion! 
Linking to the list makes it more maintainable.

Comment (ryo-manba) [reply lib/standalone.mjs]: Fixed in this commit. 
[update message](https://github.com/stylelint/stylelint/pull/8115/commits/306b314cbd1e843feec346a8035c3063354e1e2c)

File: lib/standalone.mjs [hunk 2]
```diff
@@ -111,7 +111,7 @@ export default async function standalone({
 		emitDeprecationWarning(
 			'"github" formatter is deprecated.',
 			'GITHUB_FORMATTER',
-			'Please use "stylelint-actions-formatters" instead.',
+			'See https://github.com/stylelint/awesome-stylelint/blob/main/README.md#formatters for alternative formatters.',
```
Comment (Mouvedia) [inline lib/standalone.mjs:114]:
```suggestion
			'See https://stylelint.io/awesome-stylelint#formatters for alternative formatters.',
```

**Comment by tagliala** [comment]
Thanks for this PR, and apologies for not contributing directly. I was planning to allocate some time tomorrow.

File: lib/standalone.mjs (hunk 2 above)
Comment (ryo-manba) [reply lib/standalone.mjs]: Applied in this commit.
[Replace url](https://github.com/stylelint/stylelint/pull/8115/commits/eb758e3d66bf42b4e3908171b7fb3622e2a902ef)

**Review by Mouvedia — APPROVED** [review]

File: .changeset/thick-seas-fail.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Changed: deprecation warning to suggest `@csstools/stylelint-formatter-github`
```
Comment (romainmenke) [inline .changeset/thick-seas-fail.md:5]:
```suggestion
Changed: deprecation warning to link to `https://stylelint.io/awesome-stylelint#formatters`
```

File: lib/standalone.mjs (hunk 1 above)
Comment (romainmenke) [reply lib/standalone.mjs]: Thank you!

**Review by romainmenke — APPROVED** [review]
Thank you. LGTM 👍

**Pull request merged**